DataProducts/Meteorology/Weather_v0.1.json

```
@@ -3,11 +3,12 @@
   "info": {
     "title": "Weather in metric units",
     "description": "Weather information for a given location, either current weather prediction or historical weather observations, using metric units.",
-    "version": "0.1.0"
+    "version": "0.1.2"
```

The patch version should be incremented only by one step, not two.

```suggestion
    "version": "0.1.1"
```